Kinvey/KinveyTests/MyFile.swift

```diff
@@ -6,6 +6,7 @@
 //  Copyright © 2017 Kinvey. All rights reserved.
 //
 
+import Foundation 
```

Comment:
Why do we need just the `import` statement here, but no code change?

Reply:
not sure why, but Xcode does not complain when you don't import the `Foundation` framework, but when you are compiling using the swift package manager it does complain